mythril/laser/ethereum/instructions.py

```diff
@@ -1591,12 +1605,41 @@ def create_(self, global_state: GlobalState) -> List[GlobalState]:
         :param global_state:
         :return:
         """
-        # TODO: implement me
-        state = global_state.mstate
-        state.stack.pop(), state.stack.pop(), state.stack.pop()
-        # Not supported
-        state.stack.append(0)
-        return [global_state]
+        mstate = global_state.mstate
+        environment = global_state.environment
+        world_state = global_state.world_state
+
+        call_value, mem_offset, mem_size = (
+            mstate.stack.pop(),
+            mstate.stack.pop(),
+            mstate.stack.pop(),
+        )
+
+        try:
+            call_data = get_call_data(
+                global_state,
+                util.get_concrete_int(mem_offset),
+                util.get_concrete_int(mem_offset + mem_size),
+            )
+        except TypeError:
+            log.debug("Create with symbolic length or offset. Not supported")
+            mstate.stack.append(0, 256)
+            return [global_state]
+
+        caller = environment.sender
```

Comment:
The caller should actually be the address of the active contract, not the original sender.